### PR TITLE
Upgrade proj from v9.1.0 to v9.1.1 (EL9)

### PR DESCRIPTION
**Pull request by hummeltech**
https://github.com/OSGeo/PROJ/releases

### CODE REVIEW

File: docker-compose.el9.yml [hunk 1]
```diff
@@ -172,7 +172,7 @@ x-rpmbuild:
         protobuf_c_min_version: *protobuf_c_min_version
     proj:
       image: rpmbuild-proj
-      version: &proj_version 9.1.0-1
+      version: &proj_version 9.1.1-1
       defines:
         data_version: "1.11"
```
Comment (jbronn) [inline docker-compose.el9.yml]: Upgrade the data version to [1.12](https://github.com/OSGeo/PROJ-data/releases/tag/1.12.0).

**Review by jbronn — CHANGES_REQUESTED** [review]

File: docker-compose.el9.yml (hunk 1 above)
Comment (jbronn) [reply docker-compose.el9.yml]: Fixed in 8821c17.

**Review by jbronn — APPROVED** [review]
LGTM

**Comment by hummeltech** [comment]
Awesome, thanks for the fix!

**Pull request merged**